src/emulator/storage/files.ts

```diff
@@ -610,3 +613,11 @@ export class StorageLayer {
     }
   }
 }
+
+/** Returns file separator used in given path, either '\\' or '/'. */
+function getPathSep(decodedPath: string): string {
+  // Suffices to check first separator, which occurs immediately after bucket name
+  const bucketSuffix = ".appspot.com";
```

Comment:
buckets don't have to end with .appspot.com. The guidelines for bucket naming state:

"""
Bucket names can only contain lowercase letters, numeric characters, dashes (-), underscores (_), and dots (.). Spaces are not allowed.
"""

See https://cloud.google.com/storage/docs/naming-buckets

Reply:
Done.